recipes/run-dbcan/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "run-dbcan" %}
+{% set version = "2.0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/ae/87/431b9b651544ead5b0889c39e6fddb9e1dd98e773c2ed5c229e18e063ee6/run_dbcan-2.0.11.tar.gz"
+  sha256: 79762f252c2b31a3e0dcc3b855673e9f1aac1485fabb2d94a933ab5b8b1f606c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
+
+requirements:
+  host:
+    - natsort
+    - pip
+    - python
+    - setuptools
+    - setuptools_scm_git_archive
+  run:
+    - natsort
+    - python
+    - setuptools
+    - fraggenescan
+    - prodigal
+    - diamond
+    - hmmer
+
+test:
+  commands:
+    - run_dbcan.py --help
+
+about:
+  home: "https://github.com/linnabrown/run_dbcan"
+  license: "GNU General Public v3 (GPLv3)"
+  license_family: GPL3
+  license_file: LICENSE 
+  summary: "Standalone version of dbCAN annotation tool for automated CAZyme annotation"
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```

Comment:
can you add your name here or remove it? Is there a bio.tool ID?

Reply:
No bio.tool ID